src/components/variant/ui/change-variant-button/change-variant-button.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from './change-variant-button.module.css'
+
+import { ButtonHTMLAttributes, ReactElement } from 'react'
+import cn from 'classnames'
+
+import { Icon } from '@ui/icon'
+
+interface IChangeVariantButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  direction: 'up' | 'down'
+}
+
+export const ChangeVariantButton = ({
+  className,
+  direction,
+  ...buttonProps
+}: IChangeVariantButtonProps): ReactElement => {
+  return (
+    <button
+      className={cn(styles.changeVariantButton, className)}
+      {...buttonProps}
+    >
+      <Icon
```

Comment:
And Icon should be a prop

Reply:
I think, no. If it is a `ArrowButton` it should have arrow as icon,